db/migrate/20230210104115_add_posts_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostsToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :posts, null: false, foreign_key: true
+  end
```

Comment:
According to the [RD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) a Comment should belong to a :author with foreign_key in the users table, I suggest you update this to something like:
```ruby
add_reference :likes, :author, foreign_key: { to_table: 'users' }
```